.azure-pipelines/azure-pipelines-win.yml

```diff
@@ -10,9 +10,9 @@ jobs:
   strategy:
     maxParallel: 4
     matrix:
-      win_cxx_compilervs2015vc14:
-        CONFIG: win_cxx_compilervs2015vc14
-        CONDA_BLD_PATH: C:\\bld\\
+      win_:
+        CONFIG: win_
+        CONDA_BLD_PATH: D:\\bld\\
```

Comment:
@m-kuhn  You can change this back to `C:` to get more space (we have to do this for `qgis`). Next re-render will reset it however. You may still run out of time...

Reply:
```suggestion
        CONDA_BLD_PATH: C:\\bld\\
```